requirements.txt

```diff
@@ -91,3 +91,4 @@ dbt-snowflake==1.4.0
 dbt-spark==1.4.0
 dbt-sqlserver==1.3.1
 dbt-trino==1.4.0
+dbt-clickhouse==1.4.0
```

Comment:
Can you also add this library to this file? 
https://github.com/mage-ai/mage-ai/blob/master/setup.py#L57
https://github.com/mage-ai/mage-ai/blob/master/setup.py#L119

Reply:
Sure, done! :slightly_smiling_face: